spec/reviews_controller_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+FactoryBot.define do
+  factory :restaurant do
+  end
+end
+
+
+describe  ReviewsController , :type => :controller do
+  context "GET index" do
+    it "has a 200 status code" do
+      get :index
+      expect(response.status).to eq(200)
+      expect(response.content_type).to eq "application/json"
+    end
+  end
+
+  context 'POST a new review' do
+    let(:restaurant) { FactoryBot.create(:restaurant)}
+    let(:rating) {3}
+
+    it "create a new review" do
+      comment_str = "test comment"
+      post :create, :params => {:review => {:name=>"test rev", :comment=>comment_str, :rating=>rating, :restaurant_id=>restaurant.id}}
+      expect(response.status).to eq(200)
```

Comment:
Again, splitting to small tests
it 'has a 200 status code'
it 'returns the new review'

I would not test the new rating here...since we can count on the ActiveRecord mechanism to add this review to the restaurant we intended

Since it is better to move the update_rating to the Restuarant model, this is where I would have tested this function